packages/client/components/PokerDimensionValueControl.tsx

```diff
@@ -78,62 +71,37 @@ interface Props {
   isFacilitator: boolean
   placeholder: string
   stage: PokerDimensionValueControl_stage
+  error?: {message: string | null}
+  onCompleted: () => void
+  onError: (error: Error) => void
+  onSubmitScore: () => void
+  isStale: boolean
+  isLocallyValidatedRef: MutableRefObject<boolean>
+  setCardScore: Dispatch<SetStateAction<string>>
+  cardScore: string
 }
 
 const PokerDimensionValueControl = (props: Props) => {
-  const {isFacilitator, placeholder, stage} = props
-  const {id: stageId, dimensionRef, meetingId, serviceField, task, taskId} = stage
+  const {
```

Comment:
Basically, I moved the fix field logic in the parent component so it could be used in both places we need